src/__tests__/rest/compliance/fungible/tradeWithCompliance.ts

```diff
@@ -0,0 +1,319 @@
+  });
+
+  it('should be able to create instruction for investor and blocked receiver', async () => {
+    investorBalance += 10;
+
+    const investorInstruction = await restClient.settlements.createInstruction(
+      venueId,
+      fungibleInstructionParams(ticker, issuer.did, investor.did, signerTxBase)
+    );
+
+    const blockedReceiverInstruction = await restClient.settlements.createInstruction(
+      venueId,
+      fungibleInstructionParams(ticker, issuer.did, blocked.did, {
+        options: { signer: issuer.signer, processMode: ProcessMode.Submit },
+      })
+    );
+
+    expect(investorInstruction).toEqual(
+      assertTagPresent(expect, 'settlement.addAndAffirmWithMediators')
+    );
+
+    expect(blockedReceiverInstruction).toEqual(
+      assertTagPresent(expect, 'settlement.addAndAffirmWithMediators')
+    );
+  });
+
+  it('should be able to call affirm on instruction for both receivers', async () => {
+    const { results: investorsPendingInstructions } =
+      await restClient.identities.getPendingInstructions(investor.did);
+
+    investorInstructionId = investorsPendingInstructions[0];
+
+    expect(investorInstructionId).not.toBeUndefined();
+
+    const investorAffirmResult = await restClient.settlements.affirmInstruction(
+      investorInstructionId,
+      investorTxBase
+    );
+
+    expect(investorAffirmResult).toEqual(
+      assertTagPresent(expect, 'settlement.affirmInstructionWithCount')
+    );
+
+    const { results: blockedPendingInstructions } =
+      await restClient.identities.getPendingInstructions(blocked.did);
+
+    blockedInstructionId = blockedPendingInstructions[0];
+
+    expect(blockedInstructionId).not.toBeUndefined();
+
+    const blockedAffirmResult = await restClient.settlements.affirmInstruction(
+      blockedInstructionId,
+      blockedTxBase
+    );
+
+    expect(blockedAffirmResult).toEqual(
+      assertTagPresent(expect, 'settlement.affirmInstructionWithCount')
+    );
+  });
+
+  it('should have transferred asset to investor', async () => {
+    const investorPortfolio = await restClient.portfolios.getPortfolio(investor.did, '0');
+
+    expect(investorPortfolio).toEqual(
+      expect.objectContaining({
+        assetBalances: expect.arrayContaining([
+          expect.objectContaining({ asset: ticker, free: investorBalance.toString() }),
+        ]),
+      })
+    );
+  });
+
+  it('should have not transferred the asset to a blocked receiver', async () => {
+    const blockedDidPortfolio = await restClient.portfolios.getPortfolio(blocked.did, '0');
+
+    expect(blockedDidPortfolio).toEqual(
+      expect.objectContaining({
+        assetBalances: expect.arrayContaining([
+          expect.objectContaining({ asset: ticker, free: blockedBalance.toString() }),
+        ]),
+      })
+    );
+  });
+
+  // Failing due to historical settlements
+  it.skip('should have affirmed the instruction for investor', async () => {
```

Comment:
I was expecting it to be fixed with the newer graphql. Can you try enabling these cases?